xenoglossia/builtins.py

```diff
@@ -214,6 +214,18 @@ def buttify(input, *args):
     """
     return re.sub("(?i)(butt|but)", u"\U0001F351", input)
 
+_DOTMATRIX_CHARS = ["⡮⢵", "⣟⣳", "⢎⣉", "⣏⡱", "⣟⣋", "⡯⠍", "⢎⣥", "⡗⢺", "⡇", "⢉⠏", "⡧⢎", "⣇⣀", "⡗⢺"
```

Comment:
Do you mind switching these to Unicode literals? For example, `u"⡮⢵"` That way you won't have to decode them from UTF-8 in the function, and it should work as-is on Python 3 too!